datalad/__init__.py

```diff
@@ -49,25 +61,12 @@
     os.environ['GIT_PYTHON_GIT_EXECUTABLE'] = \
         os.path.join(GitRunner._GIT_PATH, 'git')
 
-from .config import ConfigManager
-cfg = ConfigManager()
-
-from .log import lgr
-from datalad.utils import on_windows, get_encoding_info, get_envvars_info
+from datalad.utils import get_encoding_info, get_envvars_info
 
 lgr.log(5, "Instantiating ssh manager")
 from .support.sshconnector import SSHManager
 ssh_manager = SSHManager()
 atexit.register(ssh_manager.close, allow_fail=False)
-
-if on_windows and colorama is None:
-    from datalad.dochelpers import exc_str
-
-    lgr.warning(
-        "'colorama' Python module missing, terminal output may look garbled ["
-        "%s]",
-        exc_str(e))
```

Comment:
so `e` variable was eliminated from `locals()` even though above `except ... as e` was triggered?

Reply:
That pattern doesn't work in Python 3.  Try this in py2 and py3:

```python
try:
    raise ValueError("bbbbbb")
except ValueError as e:
    pass
raise e
```